transports/jms/src/main/java/org/mule/transport/jms/activemq/ActiveMQJmsConnector.java

```diff
@@ -53,6 +53,15 @@ protected void applyVendorSpecificConnectionFactoryProperties(ConnectionFactory
     {
         try
         {
+            Method getPrefetchPolicy = connectionFactory.getClass().getMethod("getPrefetchPolicy");
```

Comment:
why do you need to do this reflection stuff? this method is public (http://activemq.apache.org/maven/5.13.4/apidocs/org/apache/activemq/ActiveMQConnectionFactory.html#getPrefetchPolicy()) and this class is already coupled to ActiveMQ